packages/shared/src/components/post/PostEngagements.tsx

```diff
@@ -23,6 +23,9 @@ import { useActions } from '../../hooks';
 import { ActionType } from '../../graphql/actions';
 import { AdAsComment } from '../comments/AdAsComment';
 import { PostContentReminder } from './common/PostContentReminder';
+import { Typography, TypographyType } from '../typography/Typography';
+import { Button, ButtonIconPosition, ButtonSize } from '../buttons/Button';
+import { TimeSortIcon } from '../icons/Sort/Time';
```

Comment:
I think you forgot to add it to the icon index file.

Reply:
I added it here since it is a Sort icon too, but with different design but same name. And both exists from the guideline.

Reply:
I mean to export it from the barrel file :)